server/map_rpc_server_test.go

```diff
@@ -336,6 +336,63 @@ func TestGetSignedMapRootByRevision(t *testing.T) {
 	}
 }
 
+func TestSetLeavesEmpty(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
+	ctx := context.Background()
+
+	root := &trillian.SignedMapRoot{
+		MapRoot:   []byte("\000\001 \306h\237\020\201*\t\200\227m\2253\3308u(!f\025\225g\3545\025W\026\301A:\365=j\025\315\333#\364\266\341\026\000\000\000\000\000\000\000\000\000\000"),
```

Comment:
Where did you generate these bytes from? Given the code below I've inferred that it must commit to an empty tree at revision 0, but can we make this clearer?

Reply:
Extracted the "for testing" SMR creation into a separate function. Plus added tests that not necessarily need revision==1.